### ISSUE TITLE
TESTIMONIALS

### ISSUE THREAD

**Issue by Anushka-Singh1** [techhub-community/csoc_platform#65]
- Design this section from scratch. 
- Ensure that the images and text content are displayed properly. 
- Implement an automatic slider for testimonials with the option to navigate manually using arrow keys on either side. 
- Ensure that it works for all screen sizes.

![file_2023-11-05_09 58 59](https://github.com/techhub-community/csoc_platform/assets/98011558/d56e97b8-4649-4b07-8a85-e426c3722e89)

**Comment by DedSec2050** [comment]
I would like to work on this issue, please this to me.
@Anushka-Singh1

**Comment by arup1221** [comment]
@DedSec2050 have you completed?

**Comment by DedSec2050** [comment]
Yes Sir @arup1221 the testimonial section design is implemented only there might be a simple font size typo error
